app/models/medication.rb

```diff
@@ -1,5 +1,5 @@
 class Medication < ApplicationRecord
-  validates :brand_name, :generic_name, :product_ndc, presence: true
+  validates :brand_name, :product_ndc, presence: true
 
   has_many :user_medications
   has_many :users, through: :user_medications
```

Comment:
Do we want a medication to have access to users?  We should probably actually remove this because it could be a potential violation of privacy.

Reply:
i agree; i don't foresee any scenario in which we'd want to list users that have added a medication to their profile/account

Reply:
Getting rid of that relationship. 👍